tests/snuba/api/endpoints/test_organization_events_mep.py

```diff
@@ -2938,89 +2938,6 @@ def test_opportunity_score(self):
 
         assert meta["isMetricsData"]
 
-    def test_opportunity_score_with_fixed_weights(self):
```

Comment:
I'm assuming these tests were only relevant when the feature flag was off?

Reply:
yup!